src/nix/repl.cc

```diff
@@ -440,6 +441,7 @@ bool NixRepl::processLine(string line)
              << "  <x> = <expr>  Bind expression to variable\n"
              << "  :a <expr>     Add attributes from resulting set to scope\n"
              << "  :b <expr>     Build derivation\n"
+             << "  :e <expr>     Open the derivation in $EDITOR\n"
```

Comment:
Unrelated to this PR, but it probably makes sense to curate an (unreleased) `doc/manual/release-notes/rl-2.4.xml` that gets updated with upcoming changes as we add them.  That way @edolstra doesn't have to go through the git log and write them all himself when he wants to do a release, and users can see what unreleased changes have accumulated on master.

As an extra aside, it looks like we don't have man pages or manual documentation for the nix 2.0 CLI, so I guess this is the only place it needs to be documented?